README.md

```diff
@@ -83,6 +90,34 @@ The following options can be used to further configure the processing:
 If no option is set, the tool performs layout detection of main regions (background, text, images, separators and marginals).
 The best output quality is produced when RGB images are used as input rather than greyscale or binarized images.
 
+### Binarization
+Document Image Binarization
+
+The command-line interface for binarization of single image can be called like this:
+
+```sh
+eynollah binarization \
+  -m <path to directory containing model files> \
+  <input image> \
+  <output image>
+```
+
+and for flowing from a directory like this:
+
+```sh
+eynollah binarization \
+  -m <path to directory containing model files> \
+  -di <directory containing image files> \
+  -do <output directory>
+```
```

Comment:
I would remove this and include the directory flow in the example call above, just like it is for layout analysis.